modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,293 @@
+// Module included in the following assemblies:
+//
+// * machine_management/
+// * machine_management/
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-node-custom-partition_{context}"]
+= Optional: Adding a new {op-system} worker node with a custom `/var` partition in AWS
+
+{product-title} supports partitioning devices during installation by using machine configs that are processed during the bootstrap. However, if you use `/var` partitioning, the device name must be determined at installation and cannot be changed. This means that you cannot add different instance types as nodes if they have a different device naming schema. For example, if you configured the `/var` partition with the default AWS device name for `m4.large` instances, `dev/xvdb`, you cannot directly add an AWS `m5.large` instance, as `m5.large` instances use a `/dev/nvme1n1` device by default. The device might fail to partition due to the different naming schema.   
+
+The procedure in this section shows how to add a new {op-system-first} compute node with an instance that uses a different device name from what was configured at installation. You create a custom user data secret and configure a new machine set. These steps are specific to an AWS cluster. The principles apply to other cloud deployments also. However, the device naming schema is different for other deployments and should be determined on a per-case basis.
+
+.Procedure 
+
+. Change to the `openshift-machine-api` namespace:
```

Comment:
As I recall, ISG likes to introduce the first step for command line things explicitly, e.g. "On a command line..."